docs/openfaas-pro/iam/function-authentication.md

```diff
@@ -0,0 +1,272 @@
+# Function Authentication
+
+With OpenFaaS Identity and Access Management (IAM) you can secure function endpoints without having to write any additional code. Access permissions for functions can be configured using IAM Roles and Policies.
+
```

Comment:
Add a `!!! note` saying that function Authentication is in technical preview.

Reply:
A note was added.